keymanapp/keyman
# chore(windows): add bcp47 code and tips to diagnostic

Fixes: #12557
Add HKCU\Control Panel\International to diagnostic report the XML was already there. I just needed to modify the XSLT to display the information.

Here are the two tables
![image](https://github.com/user-attachments/assets/4762c914-d7f7-435a-8813-695ad493da3f)

# User Testing

# TEST_DIAGNOSTIC_TABLES

1. Install Keyman attached to PR.
2. Open the Keyman Configuration choose the support tab
3. Run the diagnostics
4. When the Keyman System Information appears Select `Options->Advanced View`
Expected result Select the Language Environment tab and observe that tables
`User Profile Options`  and `BCP47 Codes`
